build.gradle

```diff
@@ -220,53 +221,63 @@ artifacts {
 }
 
 task copyToLibs(type: Copy) {
-	into "$buildDir/libs"
-	from configurations.runtime
-    }
+    into "$buildDir/libs"
+    from configurations.runtime
+}
 
-task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar]) {
+task copyToRoot(type: Copy, dependsOn: [copyToLibs, jar, converterJar,
+        'unixLineEndings']) {
     outputs.files.setFrom(file("$buildDir/libs/pcgen-${version}.jar"))
 
-	from "$buildDir/libs/pcgen-${version}.jar"
-	from "$buildDir/libs/pcgen-${version}-batch-convert.jar"
-	from "$buildDir/release/pcgen.exe"
-	from "$projectDir/code/pcgen.sh"
-	into "$projectDir"
- 	rename "(.+)-$version(.+)", '$1$2'
+    from "$buildDir/libs/pcgen-${version}.jar"
+    from "$buildDir/libs/pcgen-${version}-batch-convert.jar"
+    from "$buildDir/release/pcgen.exe"
+    from "$projectDir/code/pcgen.sh"
+    into "$projectDir"
+    rename "(.+)-$version(.+)", '$1$2'
+}
+
+//Install the tools from: http://www.thefreecountry.com/tofrodos/
+task unixLineEndings( type: Exec) {
+    if(Os.isFamily(Os.FAMILY_WINDOWS)){
+        commandLine "cmd", "/c", "fromdos $projectDir/code/pcgen.sh"
```

Comment:
This still isn't quite right, 'fromdos' is not a native executable on windows. I think you need to specify the file path of fromdos.exe for this to work. I don't see you including fromdox.exe in this commit so that likely needs to be added too.